src/commands/fun/diceroll.ts

```diff
@@ -0,0 +1,28 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = 'Get a random number within a user-specified range (min and max inclusive). If no range is specified by the user, defaults to a range from 1 to 6. Also works for negative numbers!';
+	usage = '<min #> | <max #>';
```

Comment:
The convention is to use square brackets for optional args and pointy ones for required ones. Judging by `argParser()`, these should be square brackets.

Reply:
Fixed now, thank you!